system/services/email/EmailTemplateService.cfc

```diff
@@ -146,7 +146,7 @@ component {
 				, unsubscribeLink    = unsubscribeLink
 				, viewOnline         = viewOnline
 			);
-			message.htmlBody = $renderContent( renderer="richeditor", data=preppedHtml.html, context="email", args={ styles=preppedHtml.styles } );
```

Comment:
It is important to do the `$renderContent` _after_ `_prepareHtml` because `_prepareHtml` can be _cached_ without dynamic content. We then render richeditor content which can add dynamic content.

I think this will need revisiting.